evals/registry/evals/make-me-say.yaml

```diff
@@ -1,30 +1,30 @@
 make-me-say:
-  id: make-me-say.medium-and-hard.v0
+  id: make-me-say.medium-and-hard.v1
```

Comment:
We should leave these at `v0` since this doesn't constitute a new version (whereas a change to the games rules would).

Reply:
This was based on the versioning in [MakeMePay](https://github.com/openai/evals/tree/main/evals/elsuite/make_me_pay), but will remove if that is more appropriate here.